lib/ansible/modules/files/replace.py

```diff
@@ -57,21 +57,25 @@
       - The string to replace regexp matches.
       - May contain backreferences that will get expanded with the regexp capture groups if the regexp matches.
       - If not set, matches are removed entirely.
+      - Backreferences can be used ambiguously like C(\\1), or explicitly
+        like C(\\g<1>).
```

Comment:
I'm not certain if the backslash escaping here is necessary (eg `C(\\1)` versus `C(\1)`)

Reply:
No, backslash escaping is not needed here.